lib/Controller/MessagesController.php

```diff
@@ -132,7 +131,8 @@ public function __construct(string $appName,
 								IURLGenerator $urlGenerator,
 								ContentSecurityPolicyNonceManager $nonceManager,
 								ITrustedSenderService $trustedSenderService,
-								IMailTransmission $mailTransmission) {
+								IMailTransmission $mailTransmission,
```

Comment:
let's revert those two lines. the trailing comma is problematic on php7.2 (an old bug installed new Mail in the past, hence we try to keep php7.2 for a little longer)